### ISSUE TITLE
Expand Finnish data queries

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-Data#228]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/scribe-org/Scribe-Data/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow Scribe-Data's [Code of Conduct](https://github.com/scribe-org/Scribe-Data/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

This issue would look into expanding the [src/scribe_data/language_data_extraction/Finnish](https://github.com/scribe-org/Scribe-Data/blob/main/src/scribe_data/language_data_extraction/Finnish) files with as much data as are possible from the current data on Wikidata. We can use code for getting data from other languages, and from there we can check Finnish data on Wikidata for what conjugations are available. We can then expand the query with optional selections of certain forms as is done in other SPARQL queries. The query can be tried on the [Wikidata Query Service UI](https://query.wikidata.org/) during development :)

Data types to include:
- [x] Nouns
- [x] Verbs
- [x] Adjectives
- [x] Adverbs
- [x] Prepositions
- [x] Emoji keywords

### Contribution

Happy to support here as needed! I can answer questions from potential contributors and also review the PR :)

**Comment by Ebeleokolo** [comment]
Hello @andrewtavis, I would like to work on this task. Is it ok to assign this task to me? Thank you

**Comment by andrewtavis** [comment]
Sounds good, @Ebeleokolo! Let us know if you need any support :)

**Comment by andrewtavis** [comment]
Just added a list of data types that we want to include to this issue :) Have marked those that are already done or have PRs open, and we can work on the others 😊 If the data type can't work, then we can move to the others and open up specific issues later :)

**Comment by andrewtavis** [comment]
Feel free to communicate on what you're working on, @VNW22 and @Ebeleokolo :) Would be great if each of you raised PRs for this issue 😊

**Comment by VNW22** [comment]
Hey @Ebeleokolo, what have you been working on?  coz I have already done most of it. Let me know before I do a PR so that I dont post what you've already worked on.

**Comment by andrewtavis** [comment]
You can also do the full PR and we can find @Ebeleokolo another issue, @VNW22 :) No use in the work you've already done going to waste.

**Comment by Ebeleokolo** [comment]
hello @VNW22 I am acyually working on the verb query

**Comment by andrewtavis** [comment]
Great, looks like @VNW22 didn't include that in #348, so it's still for you to do. Let us know if we can support :)

**Comment by andrewtavis** [comment]
#348 is in, so we're just waiting on verbs here :) Let us know how if you need help, @Ebeleokolo! 😊

**Comment by VNW22** [comment]
@Ebeleokolo, if you need any help with the verb query, feel free to reach out—I’d be happy to assist!

**Comment by andrewtavis** [comment]
Looks like we're all done here :) Thanks @VNW22 and @Ebeleokolo! 😊
